cartography/intel/azure/aks.py

```diff
@@ -119,7 +119,33 @@ def _load_aks_tx(tx: neo4j.Transaction, subscription_id: str, aks_list: List[Dic
         SUBSCRIPTION_ID=subscription_id,
         update_tag=update_tag,
     )
+    for aks in aks_list:
+        if aks.get('resource_group'):
+            _attach_resource_group_aks(tx,aks.get('id'),aks.get('resource_group'),update_tag)
+        else:
+            x = aks['id'].split('/')
```

Comment:
let's move this logic of reading resource group from id to a common place and use that everywhere. instead of writing the same logic in all files. would be easier to maintain going forward